js/cm-api-v2/GetVariantsWithComponentsOfType.js

```diff
@@ -0,0 +1,21 @@
+// DocSection: cm_api_v2_get_variants_with_components_of_type
+// Tip: Find more about JS/TS SDKs at https://docs.kontent.ai/javascript
+// Using ES6 syntax
+import { ManagementClient } from '@kentico/kontent-management';
+
+const client = new ManagementClient({
+  projectId: '<YOUR_PROJECT_ID>',
+  apiKey: '<YOUR_API_KEY>'
+});
+
+client.listLanguageVariantsOfContentTypeWithComponents()
```

Comment:
The identifiers should match the existing ones for the endpoint. See request sample at https://docs.kontent.ai/reference/management-api-v2#operation/list-language-variants-containing-components-by-type.